web/gridt/schemas.py

```diff
@@ -46,7 +46,21 @@ def validate_token(self, value):
         except jwt.InvalidTokenError:
             raise ValidationError("Invalid token.")
 
-            
-class ChangeEmailSchema(Schema):
+
+class RequestEmailChangeSchema(Schema):
     password = fields.Str(required=True)
     new_email = fields.Email(required=True)
+
+
```

Comment:
Is this not a repeat of another schema? Maybe you can join them and name it more generally?

Reply:
As it stands right now, none of the schemas behave the same, because the endpoints need to behave differently. User needs to be authenticated for /user/change_email/request (RequestChangeEmailSchema), but not for /user/reset_password/request (RequestPasswordResetSchema).

In ResetPasswordSchema, the password needs to be included separately from the token, in ChangeEmailSchema, the new e-mail is already included in the token.